manim/mobject/coordinate_systems.py

```diff
@@ -495,3 +503,321 @@ def get_coordinate_labels(self, *numbers, **kwargs):
+            - ``"degrees"``: :math:`\\frac{1}{36}`
+            - ``"gradians"``: :math:`\\frac{1}{40}`
+            - ``None``: :math:`1`
+
+    radius_line_frequency : :class:`float`, optional
+        The frequency of faded lines in the radius.
+
+    azimuth_units : Optional[:class:`str`], optional
+        Specifies a default labelling system for the azimuth. Choices are:
+
+            - ``"PI radians"``: Fractional labels in the interval :math:`\\left[0, 2\\pi\\right]` with :math:`\\pi` as a constant.
+
+              .. manim:: PolarPlanePIradians
+                    :ref_classes: PolarPlane
+                    :save_last_frame:
+
+                    polarplot = PolarPlane(azimuth_units="PI radians").scale(0.4)
+                    polarplot.add_coordinates()
+
+                    class PolarPlanePIradians(Scene):
+                        def construct(self):
+                            self.add(polarplot)
+
+            - ``"TAU radians"``: Fractional labels in the interval :math:`\\left[0, \\tau\\right]` (where :math:`\\tau = 2\\pi`) with :math:`\\tau` as a constant.
+
+              .. manim:: PolarPlaneTAUradians
+                    :ref_classes: PolarPlane
+                    :save_last_frame:
+
+                    polarplot = PolarPlane(azimuth_units="TAU radians").scale(0.4)
+                    polarplot.add_coordinates()
+
+                    class PolarPlaneTAUradians(Scene):
+                        def construct(self):
+                            self.add(polarplot)
+
+            - ``"degrees"``: Decimal labels in the interval :math:`\\left[0, 360\\right]` with a degree (:math:`^{\\circ}`) symbol.
+
+              .. manim:: PolarPlaneDegrees
+                    :ref_classes: PolarPlane
+                    :save_last_frame:
+
+                    polarplot = PolarPlane(azimuth_units="degrees").scale(0.4)
+                    polarplot.add_coordinates()
+
+                    class PolarPlaneDegrees(Scene):
+                        def construct(self):
+                            self.add(polarplot)
+
+            - ``"gradians"``: Decimal lables in the interval :math:`\\left[0, 400\\right]` with a superscript "g" (:math:`^{g}`).
+
+              .. manim:: PolarPlaneGradians
+                    :ref_classes: PolarPlane
+                    :save_last_frame:
+
+                    polarplot = PolarPlane(azimuth_units="gradians").scale(0.4)
+                    polarplot.add_coordinates()
+
+                    class PolarPlaneGradians(Scene):
+                        def construct(self):
+                            self.add(polarplot)
+
+    azimuth_offset : Optional[:class:`float`], optional
+        The angle offset of the azimuth labels, expressed in radians. If ``None`` is specified then it will
+        use the default specified by ``azimuth_units``.
+
+            - ``"PI radians"`` or ``"TAU radians"`` or ``None``: :math:`0`
+            - ``"degrees"`` or ``"gradians"``: :math:`\\frac{\\pi}{2}`
```

Comment:
I feel that this is a pretty weird inconsistency and would actually prefer if the default offset was 0 in all cases.

Reply:
This is based on the convention of 0 degrees and gradians being oriented to "north" which is of course typically the vertical axis on most planar projections (maps etc.).

Reply:
https://en.m.wikipedia.org/wiki/Polar_coordinate_system says that this is mainly true for navigation, but not in general mathematics. (Also searching the web for polar planes in degrees yields most results with the orientation to the right.)

Reply:
Fair enough, manim is primarily a mathematics package so I understand your point and will change the default. It's obviously easily changed if someone needs to use it for illustrating navigational data.